### ISSUE TITLE
Image files

### ISSUE THREAD

**Issue by paanoop** [kochi-noders/member-portal#2]
We need the following images:
* Logo: It's needed for the kochi-noders repository as well as in the web-application.
* fav icon: icon format of the logo.

I've something in mind for the logo. KNJS written in a beautiful manner. Represents Kochi Noders and KeralaJS.

Find out some one who can do image is important :+1:

**Comment by renjupaul** [comment]
yeah. we will find some one. Nikhil will be a help for that. He has guys who can do it.

**Comment by paanoop** [comment]
Well, this is lying dormant now!
